pkg/testframework/framework.go

```diff
@@ -134,12 +156,31 @@ func IsResourceAvailable(errChan chan error, client *Clientset, resource string)
 	errChan <- err
 }
 
+func IsResourceAvailable_(client *Clientset, resource string) error {
```

Comment:
Another way to do the same thing?

Reply:
yeah... sorry for that ... forgot to clean this :D